linera-service/src/linera.rs

```diff
@@ -894,6 +895,36 @@ enum ProjectCommand {
     ///
     /// Equivalent to running `cargo test` with the appropriate test runner.
     Test { path: Option<PathBuf> },
+    /// Build and publish a Linera project.
+    PublishAndCreate {
+        /// The path of the root of the Linera project.
+        /// Defaults to CWD if unspecified.
```

Comment:
```suggestion
        /// Defaults to current working directory if unspecified.
```
(Or "current directory"?)